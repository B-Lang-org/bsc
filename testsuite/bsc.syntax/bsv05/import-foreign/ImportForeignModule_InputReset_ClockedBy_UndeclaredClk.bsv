
import "BVI" MOD =
module mkMod () ;

   default_clock no_clock ;
   no_reset ;

   input_clock clk (CLK) <- exposeCurrentClock ;

   // use the wrong name here
   input_reset rst (RSTN) clocked_by (foo) = noReset;

endmodule

