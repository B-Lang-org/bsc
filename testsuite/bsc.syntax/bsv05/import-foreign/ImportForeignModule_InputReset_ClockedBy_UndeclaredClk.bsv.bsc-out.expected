checking package dependencies
Error: "ImportForeignModule_InputReset_ClockedBy_UndeclaredClk.bsv", line 11, column 39: (P0134)
  undeclared clock `foo'
